data/vector.go

```diff
@@ -447,6 +447,58 @@ func NewFieldFromFieldType(p FieldType, n int) *Field {
 	return f
 }
 
+// NewNullableFieldFromFieldType creates a new nullable Field of the given pType of length n.
```

Comment:
There's no type or parameter called `pType`:
```suggestion
// NewNullableFieldFromFieldType creates a new nullable Field of the given FieldType of length n.
```

Reply:
Used to be called PrimitiveType .. or something similar. Might be traces of pType to cleanup elsewhere.